packages/react-api/src/Api.tsx

```diff
@@ -155,13 +155,15 @@ async function loadOnReady (api: ApiPromise, store: KeyringStore | undefined, ty
 
   setDeriveCache(api.genesisHash.toHex(), deriveMapCache);
 
+  const isEthereum:boolean = (api.runtimeVersion.specName.eq('node-moonbeam') || api.runtimeVersion.specName.eq('moonbase-alphanet'));
```

Comment:
This is not quite scalable. It was already horrible with Käufer the one check, but think it will get completely out of control with multiples. 

This is ok as a „get it going“ (for this PR and now) but probably need to move something like this into api-config, where all the chain/spec config sits.

Reply:
Ok sounds good. Should I create an issue for this in your repo? I can assign it to myself if you want

Reply:
It is fine, I can also do so. (Should be a quick one, just don’t want to lose track)